server/src/services/package.service.ts

```diff
@@ -170,6 +175,20 @@ class PackageService {
         return validPackages;
     };
 
+    transformFreeTextIntoPackagesGenerationParams = async (freeText: string): Promise<PackagesGenerationParamsWithStringDates> => {
```

Comment:
```suggestion
    transformFreeTextIntoPackagesGenerationParams = async (freeText: string) => {
```
let `typescript` infer the return type on its own